core/site_scons/site_tools/micropython/__init__.py

```diff
@@ -51,6 +52,7 @@ def generate_frozen_module(source, target, env, for_signature):
         sed_scripts = " ".join(
             [
                 rf"-e 's/utils\.MODEL_IS_T2B1/{is_t2b1}/g'",
+                rf"-e 's/utils\.MODEL_IS_T3T1/{is_t3t1}/g'",
```

Comment:
let's not introduce more situation-specific symbols

instead change those to work like the `LAYOUT` below, iow, the replacement should be looking for `utils.MODEL == "T2B1"`

(ideally at some point in the future this sed block should instead be something more like libcst parsing the sources and modifying the symbols, so that, e.g., you can catch `utils.MODEL in ("T2B1", "T2T1")` which is basically impossible here.
but that's for the future)